app/setup_handlers.go

```diff
@@ -1,34 +1,38 @@
 package app
 
 import (
-	tokenregistrymigrations "github.com/Sifchain/sifnode/x/tokenregistry/migrations"
-	tokenregistrytypes "github.com/Sifchain/sifnode/x/tokenregistry/types"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/mint"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
-const upgradeNameV095 = "0.9.5"
+const upgradeNameV096 = "0.9.6"
 
 func SetupHandlers(app *SifchainApp) {
-	SetupHandlersForV095(app)
+	SetupHandlersForMint(app)
 }
 
-func SetupHandlersForV095(app *SifchainApp) {
-	app.UpgradeKeeper.SetUpgradeHandler("0.9.5", func(ctx sdk.Context, plan types.Plan) {
-		app.Logger().Info("Running upgrade handler for " + upgradeNameV095 + " with new store " + tokenregistrytypes.StoreKey)
-		// Install initial token registry entries for non-ibc tokens.
-		tokenregistrymigrations.Init(ctx, app.TokenRegistryKeeper)
+func SetupHandlersForMint(app *SifchainApp) {
+	app.UpgradeKeeper.SetUpgradeHandler("0.9.6", func(ctx sdk.Context, plan types.Plan) {
```

Comment:
we don't need this now, but shouldn't the first argument refer to the const rather than a hard-coded version string?

Reply:
@marshsif Yes, it should.